sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -251,29 +251,33 @@ def get_all_regions_instance_types_df(regions: Set[str]) -> pd.DataFrame:
 
 
 # Fetch Images
-_GPU_TO_IMAGE_DATE = {
-    # https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;v=3;search=:64,:Ubuntu%2020,:Deep%20Learning%20AMI%20GPU%20PyTorch # pylint: disable=line-too-long
-    # Current AMIs:
-    # Deep Learning AMI GPU PyTorch 1.10.0 (Ubuntu 20.04) 20220308
-    #   Nvidia driver: 510.47.03, CUDA Version: 11.6 (does not support torch==1.13.0+cu117)
-    #
-    # Use a list to fallback to newer AMI, as some regions like ap-southeast-3 does not have
-    # the older AMI.
-    'gpu': ['20220308', '20221101'],
-    # Deep Learning AMI GPU PyTorch 1.10.0 (Ubuntu 20.04) 20211208
-    # Downgrade the AMI for K80 due as it is only compatible with
-    # NVIDIA driver lower than 470.
-    'k80': ['20211208']
-}
-_UBUNTU_VERSION = ['18.04', '20.04']
-
-
-def _fetch_image_id(region: str, ubuntu_version: str,
-                    creation_date: str) -> Optional[str]:
+# https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;v=3;search=:64,:Ubuntu%2020,:Deep%20Learning%20AMI%20GPU%20PyTorch # pylint: disable=line-too-long
+# Current AMIs (we have to use different PyTorch versions for different OS as Ubuntu 18.04
+# does not have the latest PyTorch version):
```

Comment:
Do you know why? And can't we drop the support for 18.04?

Reply:
The main reason is that 18.04 is replaced by 20.04 from the perspective of AWS. The end of life of that system for support is in April 2023 ([reference](https://computing.cs.cmu.edu/news/2022/eol-ubuntu-1804)).

We may still want the support for 18.04, as some of our users explicitly requested that OS version (also the main reason we added the `images.csv`). Since the 18.04 will not be used unless the user explicitly specifies image_id to be `skypilot:gpu-ubuntu-1804`, I think the inconsistency here might be fine. Wdyt?

Reply:
Got it. Let's keep them. Thanks for the detailed explanation!